nginx.conf

```diff
@@ -29,8 +29,6 @@ http {
     lua_shared_dict auto_ssl 1m;
     lua_shared_dict auto_ssl_settings 64k;
     lua_shared_dict page_cache 15m;
-    ssl_session_cache   shared:SSL:10m;
```

Comment:
moved to SSL shared config file -- there's no functional difference, just logically grouping settings.